ci-operator/config/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases.yaml

```diff
@@ -0,0 +1,151 @@
+base_images:
+  assisted-installer:
+    name: assisted
+    namespace: ocp-kni
+    tag: assisted-installer
+  assisted-installer-agent:
+    name: assisted
+    namespace: ocp-kni
+    tag: assisted-installer-agent
+  assisted-installer-controller:
+    name: assisted
+    namespace: ocp-kni
+    tag: assisted-installer-controller
+  assisted-test-infra:
+    name: assisted-test
+    namespace: ocp-kni
+    tag: assisted-test-infra
+  coreos_centos:
+    name: coreos-centos8-buildroot
+    namespace: coreos
+    tag: latest
+  coreos_centos_stream8:
+    name: stream8
+    namespace: coreos
+    tag: "8"
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openshift_release_golang-1.15:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+  openshift_release_golang-1.16:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
```

Comment:
it's weird we need them both
@YuviGold / @lranjbar why we have those both on the assisted-service jobs? (for both master and ocm-2.3)
anyway, not a blocker for this PR

Reply:
@osherdp  Ill leave these in. If it is not needed, We can remove them from all needed branches together. Dont wwant to add other configs in this PR.